src/app/modules/shared/components/breadcrumb/breadcrumb.component.html

```diff
@@ -1,14 +1,24 @@
-<div class="flex items-center position-relative mb-8 mt-1">
-  <div class="m-0 text-lg capitalize text-white text-center">Dashboard</div>
-  <div class="m-0 ml-4 text-hint">|</div>
-  <mat-icon 
-    aria-hidden="false" 
-    aria-label="Example home icon"
-    class="text-center ml-2 text-primary"
-  >
-    home
-  </mat-icon>
-  <span class="capitalize text-muted ml-4">
-    Gains &amp; Losses
-  </span>
-</div>
\ No newline at end of file
+<nav class="flex items-center position-relative mb-8 mt-1" *ngIf="(breadcrumbs$ | async) as breadcrumbs">
```

Comment:
Async pipe here makes it super easy to handle rxjs logic, without needing to subscribe in the component.ts file